vault/seal.go

```diff
@@ -76,9 +78,10 @@ type Seal interface {
 }
 
 type defaultSeal struct {
-	access seal.Access
-	config atomic.Value
-	core   *Core
+	access     seal.Access
+	config     atomic.Value
+	core       *Core
+	metaPrefix string
```

Comment:
Not sure if I suggested this elsewhere, but not sure I like the whole `metaPrefix` thing again here, but that's fine.